lib/exq/redis/script.ex

```diff
@@ -51,6 +51,38 @@ defmodule Exq.Redis.Script do
         return {1, conflict_jid}
       end
       """),
+    enqueue_all:
+      Prepare.script("""
+      local schedule_queue = KEYS[1]
+      local i = 1
+      local result = {}
+
+      while i < #(KEYS) do
+        local unique_key = KEYS[i + 1]
+        local jid        = ARGV[(i - 1) * 4 + 1]
+        local score      = tonumber(ARGV[(i - 1) * 4 + 2])
+        local job        = ARGV[(i - 1) * 4 + 3]
+        local unlocks_in = tonumber(ARGV[(i - 1) * 4 + 4])
+        local unlocked   = true
+        local conflict_jid = nil
+
+        if unlocks_in then
+          unlocked = redis.call("set", unique_key, jid, "px", unlocks_in, "nx")
+        end
+
+        if unlocked then
+          redis.call('ZADD', schedule_queue, score, job)
```

Comment:
There are two types of enqueue,
1) schedule in the future (enqueue_at)
2) enqueue now (enqueue)

ZADD is used for scheduling in the future, SADD/LPUSH is used for enqueue now

you might want to change it to SADD/LPUSH

Reply:
The current version, expect jobs to be specified like `[queue, time, worker, args, options] = job` which by default expects time and supports scheduling for future. I was thinking if you pass `DateTime.utc_now` it will practically behave the same as enqueuing them right away. Although it may not be efficient to put jobs in schedule queue if they need to be enqueued right away.

I can make the `time` optional, and use SADD/LPUSH if time is not supplied. will see if I can also improve the API a bit.

Reply:
There are two sets of apis: enqueue and enqueue_at/in. Though using `now` might seem the same, they are not implementation wise. Scheduled jobs sit in ZSET and are then picked up later by the scheduler. Look at the current behavior and use the same behavior for enqueue_all variant as well.

Reply:
```
jobs = [
  [queue_1, worker_1, args_1, options_1],
  [queue_2, worker_2, args_2, options_2],
  [queue_3, worker_3, args_3, options_3]
  ...
]
```

I think it might be better to remove the time and move it to options (in/at keys), since time is not required for normal enqueue (which would be the major use case)